src/transcript-plugin.tsx

```diff
@@ -335,6 +335,7 @@ export class TranscriptPlugin extends KalturaPlayer.core.BasePlugin {
               this._handleAttach(CloseDetachTypes.arrow);
             }}
             onJumpToSearchMatch={this._toSearchMatch}
+            upperBarManager={this.upperBarManager}
```

Comment:
I don't think we need pass entire this.upperBarManager, we can pass only one method, like
`focusPluginButton = {this.upperBarManager!.focusPluginButton(this._transcriptIcon)}`

Reply:
done

Reply:
done